drivers/base/bootvid/i386/xbox/nv2a.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * PROJECT:     ReactOS Boot video driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Header file for NVIDIA NV2A XGPU
+ * COPYRIGHT:   Copyright 2004 Ge van Geldorp
+ *              Copyright 2005 Filip Navara
```

Comment:
Not sure the two guys from above wrote the xbox nv2a support 😛

Reply:
This code is inspired by the patch @dreimer1986 attached to Jira ticket, and he said GvG once shared it with him. Filip Navara comes from xboxvmp driver code. :)

Reply:
Ah ok so I'm fine with keeping the guys :)